mibidata/mibi_image.py

```diff
@@ -342,6 +333,34 @@ def parse_aperture(value):
                 )
         return aperture
 
+
+    @staticmethod
+    def match_fov_folder(value, field, value_name='valuename',
```

Comment:
The parameters `value_name` and `field_name` seem to be required and the default values specified in the signature are never used. Can you just leave off the defaults? Or should they default to one combination of `value_name` and `field_name`?